playground/src/kataViewer.tsx

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// import "preact/debug"; // Include this line only when debugging rendering
+
+import { render } from "preact";
+import { useEffect } from "preact/hooks";
+
+import {
+  Exercise,
+  ExplainedSolutionItem,
+  Kata,
+  Lesson,
+  getAllKatas,
+} from "qsharp-lang";
+
+declare global {
+  // The below are added by the MathJax and Highlight.js scripts
+  interface Window {
+    MathJax: any;
+    hljs: any;
+  }
+}
+
+window.MathJax = {
+  loader: {
+    load: ["[tex]/color", "[tex]/braket"],
```

Comment:
The change from `physics` to `braket` here is what fixed the braket rendering issue (for some reason).

Reply:
I suggest we add a comment here describing a difference between 'bracket' in physics and 'bracket' in bracket. Also probably worth mentioning this as a guidance on your LaTeX authoring page.

Reply:
`color` and `braket` auto-load, so are only explicitly loaded here as an optimization. `physics` doesn't auto-load and just should never have been added. (See the list of packages in __bold__ [at this link](https://docs.mathjax.org/en/latest/input/tex/macros/index.html), which are those which auto-load).

I actually figured out the issue by using the MathJax demo page and then seeing that it didn't load __any__ packages. So technically the fix is just use MathJax with the default packages and don't add anything.